pkg/controller/networkaddonsconfig/networkaddonsconfig_controller.go

```diff
@@ -286,6 +287,8 @@ func (r *ReconcileNetworkAddonsConfig) Reconcile(request reconcile.Request) (rec
 		return reconcile.Result{}, err
 	}
 
+	monitoring.TrackMonitoringComponents(&networkAddonsConfig.Spec, r.clusterInfo.MonitoringAvailable)
```

Comment:
maybe should be after `deleteObjects` ? (actually same question is about `trackDeployedObjects` but not part of this PR)

https://github.com/kubevirt/cluster-network-addons-operator/pull/955/files#diff-463a5613afbb95588d747500e304c2d348c7fce8ec35117067322e7aac36a8dcR296

Reply:
I agree it's a more fitting place.. done